setenv

```diff
@@ -6,10 +6,11 @@
 
 CITADEL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))"
 
+alias c="${CITADEL_ROOT}/bin/citadel"
```

Comment:
I'd rather not have this built-in, people may want to use c for something else. If power users want this, I think they can just add it themself.

Reply:
Sure, I'll revert.